### PR TITLE
HDDS-2552. Sonar: Save and reuse Random object in GenesisUtil

### PR DESCRIPTION
**Pull request by vivekratnavel**
## What changes were proposed in this pull request?

Save and reuse Random object in GenesisUtil

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2552

## How was this patch tested?

Verified that the project compiles successfully.

### CODE REVIEW

**Comment by vivekratnavel** [comment]
@adoroszlai @dineshchitlangia @nandakumar131 Please review

File: hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/genesis/GenesisUtil.java [hunk 1]
```diff
@@ -68,6 +68,8 @@ private GenesisUtil() {
 
   private static final int DB_FILE_LEN = 7;
   private static final String TMP_DIR = "java.io.tmpdir";
+  private static final Random RANDOM = new Random();
+  private static final String SCM_CLIENT_DEFAULT_ADDRESS = "127.0.0.1:0";
```
Comment (adoroszlai) [inline hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/genesis/GenesisUtil.java]: Given that this is also used for non-client SCM and OM addresses, wouldn't something like `RANDOM_LOCAL_ADDRESS` be a better name?

Comment (vivekratnavel) [reply hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/genesis/GenesisUtil.java]: Sure. Fixed it!

**Comment by vivekratnavel** [comment]
@adoroszlai Thanks for the review and merge!

**Pull request merged**